### PR TITLE
chore(upgrade): Upgrade to PHP 7.3.10 / 7.2.23

### PR DESCRIPTION
**Pull request by BitScout**
Upgrade PHP and its utilities to the latest versions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Versions
 
 * Fix Chrome image build
 * Upgrade Golang version: 1.13.3
+* Upgrade PHP version: 7.2.24 / 7.3.10
```
Comment (mremi) [inline CHANGELOG.md]:
```suggestion
* Upgrade PHP version: 7.2.23 / 7.3.10
```

File: config.yml [hunk 2]
```diff
@@ -204,31 +204,31 @@ php:
...
       CI_HELPER_VERSION: *CI_HELPER_VERSION
       COMPOSER_VERSION: 1.9.0
-      MEMCACHED_VERSION: 3.1.3
+      MEMCACHED_VERSION: 3.1.4
       MODD_VERSION: *MODD_VERSION
       PHP_CS_FIXER_VERSION: 2.15.3
       REDIS_VERSION: 5.0.2
       SECURITY_CHECKER_VERSION: 6.0.2
-      SSH2_VERSION: 1.1.2
+      SSH2_VERSION: 1.2
```
Comment (mremi) [inline config.yml]: Nice, could you please try to uncomment the installation of ssh2 in Dockerfile (and so re-add it in `test.php`)?

Comment (BitScout) [reply config.yml]: > Could not download from "https://pecl.php.net/get/ssh2-1.2.tgz", cannot download "pecl/ssh2"

The link is on the download page, but it's dead... https://pecl.php.net/package/ssh2

Comment (mremi) [reply config.yml]: Ok thanks, so let's wait

**Pull request merged**